### ISSUE TITLE
Error Deploying attributes having Anchor tags , anchortags are missing in destination

### ISSUE THREAD

**Issue by vinay14feb** [forcedotcom/dataloader#690]
I was trying to Deploy a tile object that has data with anchor tags. The hyperlinks are not moving to destination

for example when i am moving 
<ul><li><a href=“wwww.google.com” target="_blank" style="font-size: 14px;">Sample Link</a></li></ul> is deployed as 
<ul><li>Sample Link</li></ul>

**Comment by vinay14feb** [comment]
using version 57.0.0 , not able to move anchor tags with href to destination

**Comment by vinay14feb** [comment]
Hi @ashitsalesforce  It was working fine using salesforce version 55.0 but the issue is happening with 57.0. The attribute is nForce_body_c and the type is Rich Text area

**Comment by ashitsalesforce** [comment]
Hi @vinay14feb , you are running into an issue that was caused by a fix for another issue (PR #485) and fixed in PR #649 . Download the latest version of data loader (current 58.0.2) containing the fix for your issue.

**Comment by ashitsalesforce** [comment]
Hi @vinay14feb ,

Good to hear that the issue is addressed. 

Regarding your question, I think you need to take the following steps:
Step 1: Upload image files through insert, upsert, or update of `ContentVersion` object. A sample CSV file to upload is:
```
Title,VersionData,PathOnClient
"sample1.png","/home/test/images/sample1.png","/home/test/images/sample1.png"
```
Step 2: Extract the URL of uploaded image files. Construct the SoQL query as
```
Select Title, VersionDataUrl, PathOnClient from ContentVersion where Title='sample1.png'
```
You will get the result
```
Title, VersionDataUrl, PathOnClient
"sample1.png", "https://<orgname>.file.force.com/sfc/servlet.shepherd/version/download/0984W00050X22Ag","/home/test/images/sample1.png"
```

Step 3: Extract value of `VersionDataUrl` field from the downloaded CSV file in the previous step and specify in the RichText field of the destination sObject. For example, if you have a custom field `Image__c` on Account, you may create the following CSV file for an insert operation:
```
name, Image__c
<img src=“”https://<orgname>.file.force.com/sfc/servlet.shepherd/version/download/0984W00050X22Ag”” alt=""sample1.png””></img>
```

Step 4: Upload the CSV prepared in the previous step through an insert, upsert, or update operation.
